libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -19,6 +19,15 @@ class AC_WPNav
     /// Constructor
     AC_WPNav(const AP_InertialNav& inav, const AP_AHRS_View& ahrs, AC_PosControl& pos_control, const AC_AttitudeControl& attitude_control);
 
+    // changed getter and setter of waypoint radius //////////////////////////////////////////////////////////////
+
+    void set_wp_radius(float radius) {_wp_radius_cm = radius;}
```

Comment:
The WP radius is at least 5 cm.
Config parameters are set using the .set method.

    // check _wp_radius_cm is reasonable
    _wp_radius_cm.set(MAX(_wp_radius_cm, WPNAV_WP_RADIUS_MIN));